tsconfig.build.json

```diff
@@ -3,7 +3,14 @@
     "compilerOptions": {
         "noEmit": false /* Do not emit outputs. */,
         "importHelpers": true /* Import emit helpers from 'tslib'. */,
-        "noEmitHelpers": true
+        "noEmitHelpers": true,
+        "module": "es2022",
+        "moduleResolution": "node",
```

Comment:
This differs from the tsconfig.json, probably should have the same across both?

Reply:
I think I have it sorted now. This repo specifies module and moduleResolution in three different place. tsconfig.json, tsconfig.build.json and within the build scripts in package.json. There is a dual build setup to support both CJS and ES Modules.

I don't spend much time in the plumbing of JS projects these days so getting a combination that worked was a bit of a fiddle. If I've done anything wrong or foolish, let me know.